### PR TITLE
Update integration tests composer cache

### PR DESCRIPTION
**Pull request by MorrisJobke**
Just a fresh image with updated composer dependencies.

### CODE REVIEW

**Review by rullzer — APPROVED** [review]

**Comment by MorrisJobke** [comment]
cc @nickvergessen @LukasReschke

File: .drone.yml [hunk 1]
```diff
@@ -454,21 +454,21 @@ pipeline:
...
-      image: nextcloudci/php7.0:php7.0-7
+      image: nextcloudci/integration-php7.0:integration-php7.0-4
       commands:
         - tests/acceptance/run-local.sh --timeout-multiplier 10 allow-git-repository-modifications features/access-levels.feature
       when:
         matrix:
           TESTS-ACCEPTANCE: access-levels
   acceptance-app-files:
-      image: nextcloudci/php7.0:php7.0-7
+      image: nextcloudci/integration-php7.0:integration-php7.0-4
```
Comment (nickvergessen) [inline .drone.yml]: this test fails now

Comment (danxuliu) [inline .drone.yml]: The failures will go away when this pull request is merged into master; they are not caused by the changes in this pull request, but by an outdated test in the master branch at the time it was forked, which was already fixed in #5009.

**Comment by MorrisJobke** [comment]
@nickvergessen @danxuliu Is this okay to be merged?

**Review by LukasReschke — APPROVED** [review]
Rebased upon master. Let's see how that goes.

**Review by danxuliu — APPROVED** [review]
> @danxuliu Is this okay to be merged?

After the rebase one of the integration tests failed, but just due to the maximum build time being exceeded, which is not related to these changes, so 👍

**Pull request merged**